spec/controllers/api/v1/slack_controller_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+describe Api::V1::SlackController, type: :controller do
+  describe 'GET #get_users' do
+    let(:slack) { instance_double('SlackService') }
+    let(:slack_user) do
+      double({
+        deleted: true,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.